arviz/tests/base_tests/test_stats.py

```diff
@@ -47,6 +47,35 @@ def test_hpd():
     assert_array_almost_equal(interval, [-1.88, 1.88], 2)
 
 
+def test_hpd_2darray():
+    normal_sample = np.random.randn(12000, 5)
+    result = hpd(normal_sample)
+    assert result.shape == (5, 2,)
+
+
+def test_hpd_multidimension():
+    normal_sample = np.random.randn(12000, 10, 3)
+    result = hpd(normal_sample)
+    assert result.shape == (10, 3, 2,)
+
+
+def test_hpd_idata():
```

Comment:
These tests should use `data` instead of `normal_sample`

Reply:
you can also use the fixture instead of calling load_arviz_data: https://github.com/arviz-devs/arviz/blob/master/arviz/tests/base_tests/test_stats.py#L107